### ISSUE TITLE
Tabbar labels not updating according to set size in Settings

### ISSUE THREAD

**Issue by mkoorn** [uias/Tabman#557]
### New Issue Checklist

- [v ] Updated Tabman to the latest version
- [v] I took a look at the [Troubleshooting Guide](https://github.com/uias/Tabman/blob/main/Docs/Troubleshooting.md)
- [v ] I searched for [existing GitHub issues](https://github.com/uias/Tabman/issues)

### Issue Description

I am trying to get the label text sizes to update when the user changes in Settings->Accessibility->Display & Text size

Any idea how to force this?

**Comment by msaps** [comment]
@mkoorn the current built-in components in Tabman don't support Dynamic Type - so this is why you won't see the label font changing with the system setting.

However this is going to be resolved with #560 which adds support for Dynamic Type 😄 

You can enable it on the buttons by doing the following:

```swift
bar.buttons.customize {
    $0.adjustsFontForContentSizeCategory = true
}
```

The buttons now default to using a font with a specified text style that will support Dynamic Type when `adjustsFontForContentSizeCategory` but you can of course set a custom font that supports this. It also requires iOS 11.

**Comment by msaps** [comment]
Now available in [2.11.0](https://github.com/uias/Tabman/releases/tag/2.11.0) 🚀

**Comment by mkoorn** [comment]
Hi @msaps, thanks for the update.
I'm now revisiting this since supporting dynamic type was postponed for me.

I still have an issue.
The problem is that I am using a custom font added to the application.

The font is for example created by:
`UIFont(name: "fontName", size: 14)`
then the scaledFont is created with:
`UIFontMetrics(forTextStyle: textStyle).scaledFont(for: font)`

The problem is I think in:
`reloadTextLayerForCurrentFont`

Any ideas how to support custom dynamic fonts?
